src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -36,6 +36,8 @@
 protectable_item_name_type_help = """Specify the resource name to be protected by Azure Backup service."""
 backup_type_help = """'Full, Differential, Log, Copy-only-full' for backup Item type 'MSSQL'. 'Full, Differential' for backup item type 'SAPHANA'."""
 retain_until_help = """The date until which this backed up copy will be available for retrieval, in UTC (d-m-Y). For SAPHANA and SQL workload, retain-until parameter value will be overridden by the underlying policy."""
+diskslist_help = """List of disks to be excluded or included."""
```

Comment:
Usually Azure CLI supports both name or ID of resource.

Reply:
This is not a azure managed disk ID. This is the LUN of that disk in the context of that backed up VM. Hence this is list of LUNs.